inference-engine/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ addIeTarget(
 
 target_include_directories(${TARGET_NAME} PUBLIC
         ${PUBLIC_HEADERS_DIR}
+        ${IE_MAIN_SOURCE_DIR}/src/vpu/graph_transformer/include
```

Comment:
Why we add this lib? And Please move construction: `INCLUDE_DIRECTORIES` in addIeTarget

Reply:
This is necessary to include the "vpu/private_plugin_config" .
Ok, I'll make a change.